client/components/navHeader.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import M from 'materialize-css';
+
+export default class NavHeader extends React.Component {
+  componentDidMount() {
+    const dropdowns = document.querySelectorAll('.dropdown-trigger');
+
+    const options = {
+      inDuration: 300,
+      outDuration: 300,
+      hover: true,
+      coverTrigger: false
+    };
+
+    M.Dropdown.init(dropdowns, options);
```

Comment:
In React, we never query the DOM like this. Instead, use a `ref` to access the element.

https://reactjs.org/docs/refs-and-the-dom.html#creating-refs